samtranslator/model/apigatewayv2.py

```diff
@@ -168,10 +168,18 @@ def _validate_lambda_authorizer(self):
                 self.api_logical_id, self.name + " Lambda Authorizer must define 'AuthorizerPayloadFormatVersion'."
             )
 
-        if self.identity and not isinstance(self.identity, dict):
-            raise InvalidResourceException(
-                self.api_logical_id, self.name + " Lambda Authorizer property 'identity' is of invalid type."
-            )
+        if self.identity:
+            if not isinstance(self.identity, dict):
+                raise InvalidResourceException(
+                    self.api_logical_id, self.name + " Lambda Authorizer property 'identity' is of invalid type."
+                )
+            if self.identity.get("Headers"):
+                headers = self.identity.get("Headers")
```

Comment:
To prevent the duplication:

```suggestion
            headers = self.identity.get("Headers")
            if headers:
```